lib/tasks/bus_stop_field.rb

```diff
@@ -0,0 +1,68 @@
+        { name: 'Bolt on base', column: :bolt_on_base, field_type: :boolean, description: 'Is this a break away pole?' },
+        { name: 'Stop sticker', column: :stop_sticker, field_type: :choice, choices: ['No sticker', 'Sticker incorrect', 'Sticker correct'] },
+        { name: 'Route sticker', column: :route_sticker, field_type: :choice, choices: ['No sticker', 'Sticker incorrect', 'Sticker correct'] }
+      ],
+      'Shelter' => [
+        { name: 'Shelter', column: :shelter, field_type: :choice, choices: ['No shelter', 'PVTA shelter', 'Other shelter', 'Nearby building'], description: 'Determine if the bus stop has a shelter or not.' },
+        { name: 'Shelter type', column: :shelter_type, field_type: :choice, choices: ['Modern full', 'Modern half', 'Victorian', 'Dome', 'Wooden', 'Extra large', 'Other', 'No Shelter'], description: 'Determine the type of shelter.' },
+        { name: 'Shelter ADA compliance', column: :shelter_ada_compliance, field_type: :boolean },
+        { name: 'Shelter condition', column: :shelter_condition, field_type: :choice, choices: ['Great', 'Good', 'Fair', 'Poor', 'No shelter'], description: 'Determine the condition of the bus shelter.' },
+        { name: 'Shelter pad condition', column: :shelter_pad_condition, field_type: :choice, choices: ['Great', 'Good', 'Fair', 'Poor', 'No shelter'], description: 'Determine the condition of the bus shelter pad.' },
+        { name: 'Shelter pad material', column: :shelter_pad_material, field_type: :choice, choices: ['Asphalt', 'Concrete', 'Other', 'No shelter'], description: 'Determine the material for the shelter pad.' }
+      ],
+      'Amenity' => [
+        { name: 'Bench', column: :bench, field_type: :choice, choices: ['PVTA bench', 'Other structure', 'Other bench', 'PVTA and other bench', 'None'], description: 'Determine if the bus stop has a bench or area to sit.' },
+        { name: 'Bike rack', column: :bike_rack, field_type: :choice, choices: ['PVTA bike rack', 'Other bike rack', 'PVTA and other bike rack', 'None'], description: 'Determine if there are bike racks.' },
+        { name: 'Schedule holder', column: :schedule_holder, field_type: :choice, choices: ['On pole', 'In shelter', 'None'], description: 'Determine if there is a bus schedule or not.' },
+        { name: 'System map exists', column: :system_map_exists, field_type: :choice, choices: ['New map', 'Old map', 'None'], description: 'Determine if there is a system map at this stop.' },
+        { name: 'Trash', column: :trash, field_type: :boolean, description: "Determine if there is a trash receptacle at the stop or within 10' of the stop." }
+      ],
+      'Accessible' => [
+        { name: 'Accessible', column: :accessible, field_type: :boolean, description: 'Can this stop be safely accessed?' },
```

Comment:
Personally, I would align each field vertically, but the tradeoff is line length, so it's up to you.

Reply:
I was considering it. Perhaps I will.